friend-face-react/src/components/posts/NewPostForm.js

```diff
@@ -1,27 +1,31 @@
 import classes from "./NewPostForm.module.css";
-import { useRef } from "react";
+import { useRef, useState } from "react";
 import Card from "../ui/Card";
 
+
 function NewPostForm(props) {
   const authorInputRef = useRef();
   const dateInputRef = useRef();
   const likesInputRef = useRef();
   const contentInputRef = useRef();
+  const avatarInputRef = useRef();
+  const [colour, setColour] = useState('#0010f7');
 
   function submitHandler(event) {
```

Comment:
Could we extract this function into a serparate file, like you've done with the other handlers?